app/helpers/navbar_helper.rb

```diff
@@ -2,4 +2,12 @@ module NavbarHelper
   def active_class(link_path)
     current_page?(link_path) ? "active" : ""
   end
+
+  def settings_active_class(project)
```

Comment:
I think you don't need these helper, not sure. Could you use only `active_class(edit_project_path(project))` in this case?

https://github.com/Codeminer42/cm42-central/pull/91/files#diff-e571170d04fb0f6c6751b028b24ff1f0R15